dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -46,9 +46,9 @@ func Single(args []string) error {
 		customUfsModules := strings.Split(customUfsModuleFlag, "%")
 		for _, customUfsModule := range customUfsModules {
 			customUfsModuleFlagArray := strings.Split(customUfsModule, "|")
-			if len(customUfsModuleFlagArray) == 2 {
-				customUfsModuleFlagArray[1] = strings.ReplaceAll(customUfsModuleFlagArray[1], ",", " ")
-				ufsModules["ufs-"+customUfsModuleFlagArray[0]] = module{customUfsModuleFlagArray[0], true, customUfsModuleFlagArray[1]}
+			if len(customUfsModuleFlagArray) == 3 {
```

Comment:
also update the `custom-ufs-module` flag description. it should be a pipe-separated triplet of module name, ufs type, and its comma-separated maven arguments. the example also should be updated

Reply:
Done